ngraph/core/include/openvino/op/random_uniform.hpp

```diff
@@ -12,7 +12,7 @@ namespace v8 {
 /// \brief Tensor RandomUniform operation.
 class OPENVINO_API RandomUniform : public Op {
 public:
-    OPENVINO_RTTI_DECLARATION;
+    OPENVINO_OP("RandomUniform", "opset8", op::Op, 8);
```

Comment:
here we specify opset index 2 times:
- `"opset8"`
- `8`
Can we always specify only number while string will be auto-generated?

As far as I can see we already have information about opset in `Node`:
```c++
OPENVINO_RTTI_DEFINITION(op::v0::LRN, "LRN", 0);
```
Can we use this number to generate opset name in `serialize`?

Reply:
No because we can have situations where opset version != op version